app-api/src/main/java/org/openmrs/eip/app/receiver/ReceiverActiveMqMessagePublisher.java

```diff
@@ -65,6 +66,12 @@ private <E extends AbstractEntity> void sendSyncResponse(E sourceEntity, String
 		syncResponse.setDateSentByReceiver(LocalDateTime.now());
 		syncResponse.setMessageUuid(messageUuid);
 		
+		if (sourceEntity instanceof SyncMessage) {
```

Comment:
I don't see where you have updated tests in ReceiverActiveMqMessagePublisherTest to test these changes

Reply:
Done

Reply:
Done